docs/reference/search.asciidoc

```diff
@@ -125,9 +125,12 @@ Setting this value to `-1` resets the global search timeout to no timeout.
 [[global-search-cancellation]]
 === Search Cancellation
 
-Searches can be cancelled using standard <<task-cancellation,task cancellation>>
-mechanism. By default, a running search only checks if it is cancelled or
-not on segment boundaries, therefore the cancellation can be delayed by large
+Searches are automatically cancelled when the http connection that they came
```

Comment:
I find the sentence hard to read, maybe something like: `when the http connection used to perform the request is closed by the client` ?